src/tests/SConscript

```diff
@@ -31,6 +45,14 @@ def scons():
                                  LIBS=libs)
     denv.Install('$PREFIX/bin/', obj_ctl)
 
+    # link $PREFIX/install/lib/daos/TESTING to src/tests/ftest
+    tpath = join(denv.subst("$PREFIX"), "lib/daos/TESTING/ftest")
+    # copy ftest to $PREFIX/install/lib/daos/TESTING
+    spath = join(denv.subst("$PREFIX"), "../src/tests/ftest/")
+    # denv.InstallAs('$PREFIX/lib/daos/TESTING/ftest/', spath)
+    if not check_dir_exists(tpath):
+        os.symlink(spath, tpath)
```

Comment:
You cannot do this.  The point of `$PREFIX` (`install/` in a local installation; `/usr` in an RPM based installation) is that it supposed to be a self-contained tree of everything needed to run the tests.  You should be able to remove everything above it and it still works.

But do think of the RPM installation case.  `/usr/lib/daos/TESTING/ftest` will be a symlink to where?  The local source tree will not exist on the machine that the RPM is installed on.

Reply:
Not clear how RPM is built, but this src/tests directory is not available in RPM pkg, so this SConscript will not be executed, right?

Reply:
changed to copy instead of symlink.